### DISCUSSION TITLE
[Feature] deduplication file type preference

### DISCUSSION THREAD

**Discussion by stratacast** [immich-app/immich#11025] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

The deduplication feature is awesome. Especially the bulk deduplication. I'm not sure I like that the largest file size is selected by default. It would be nice to prefer smallest file size or even file types.

For example, as a Nextcloud -> Immich convert, I previously chose to convert all images from HEIC to jpg on upload because of the incompatibilities with the file type with Nextcloud and viewers at the time. However now, with HEIC being easier to work with now and have useful features, it would be nice to prefer the HEIC file type over jpg. This is handy for people who are importing preexisting libraries into immich, and also having to re-upload their entire camera roll from their iPhones.

### Platform

- [ ] Server
- [X] Web
- [ ] Mobile

**Comment by bo0tzz** [comment]
#10665
